flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/config/ExecutionConfigOptions.java

```diff
@@ -376,6 +379,17 @@
                                                     + "Pipelined shuffle means data will be sent to consumer tasks once produced.")
                                     .build());
 
+    // TODO: Update Description with the main upcoming change: errors thrown vs returning null
```

Comment:
We should not add too many TODOs into the code base. There is always much to do. If at all, then with a JIRA issue attached. In this case, I would simply drop the comment.

Reply:
Sure, added a comment to: https://issues.apache.org/jira/browse/FLINK-24385?focusedCommentId=17452537&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-17452537 to not forget about this.